nucypher/blockchain/economics.py

```diff
@@ -399,7 +455,13 @@ def retrieve_from_blockchain(registry: BaseContractRegistry) -> BaseEconomics:
         # Staking Escrow
         staking_parameters = list(staking_agent.staking_parameters())
         seconds_per_period = staking_parameters.pop(0)
-        staking_parameters.insert(3, seconds_per_period // 60 // 60)  # hours_per_period
+        staking_parameters.insert(6, seconds_per_period // 60 // 60)  # hours_per_period
+        minting_coefficient = staking_parameters[0]
+        lock_duration_coefficient_2 = staking_parameters[2]
+        first_phase_total_supply = staking_parameters[4]
+        first_phase_supply = first_phase_total_supply - initial_supply
+        staking_parameters[4] = first_phase_supply
```

Comment:
Can we have constants for these indices?

Reply:
I'm not sure, almost all values have names and this constant will be used only one time